blog/_posts/2016-09-20-julia-0.5-highlights.md

```diff
@@ -0,0 +1,726 @@
+---
+layout: post
+title:  Julia 0.5 Highlights
+author: Stefan Karpinski
+---
+
+Julia 0.5 is a pivotal release.
+It introduces more transformative features than any release since the first official version.
+Moreover, several of these features set the stage for even more to come in the [lead up to Julia 1.0](https://www.youtube.com/watch?v=5gXMpbY1kJY).
+In this post, we'll go through some of the major changes in 0.5, including improvements to functional programming, comprehensions, generators, arrays, strings, and more.
+
+## Functions
+
+Julia has always been a functional language in the technical sense: functions are values that can be passed to and from other functions (i.e. [higher-order functions](https://en.wikipedia.org/wiki/Higher-order_function)) and there is full support for [lambdas](https://en.wikipedia.org/wiki/Anonymous_function).
```

Comment:
Seems right to me - better to avoid calling it a functional language as any noise around that will only distract from the purpose here.